images/kafka/README.md

```diff
@@ -21,7 +21,7 @@ The image is available on `cgr.dev`:
 docker pull cgr.dev/chainguard/kafka:latest
 ```
 
-## Using NATS
+## Using KAFKA
```

Comment:
nit: I don't think we need all-caps here ("NATS" is an acronym, I believe for "Neural Autonomic Transport System")